src/jit/gentree.cpp

```diff
@@ -1740,6 +1740,35 @@ bool GenTreeCall::IsHelperCall(Compiler* compiler, unsigned helper) const
     return IsHelperCall(compiler->eeFindHelper(helper));
 }
 
+void GenTreeCall::ReplaceCallOperand(GenTree** operandUseEdge, GenTree* replacement)
```

Comment:
This method needs a header comment.

A description of what extar work thsi methdo is doing beyonf the actual  updating assignment.

It appears that it want to handle every modification of call "operands" including the gtControlExpr, gtCallCookie and gtCallAddr, although these last three don't actually require this method to do any extra work.